cumulus/polkadot-omni-node/lib/src/command.rs

```diff
@@ -226,6 +226,14 @@ pub fn run<CliConfig: crate::cli::CliConfig>(cmd_config: RunConfig) -> Result<()
 				RelayChainCli::<CliConfig>::new(runner.config(), cli.relay_chain_args.iter());
 			let collator_options = cli.run.collator_options();
 
+			if cli.experimental_use_slot_based {
```

Comment:
Maybe we should just directly kill it. This was an experimental setting, which given the name is something that could disappear any moment.

Reply:
One cute thing is that this will automatically print what to use instead. Lets keep it around for one release, I will take care of removing it.